cmd/buildah/images.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	imagesFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "omit column headings",
+		},
+		cli.BoolFlag{
+			Name:  "qq",
```

Comment:
Oh.  I was going for something along the lines of 'docker images -q', but am not opposed to renaming the flag.  Any preference?